MAUI/DataGrid/Column-types.md

```diff
@@ -0,0 +1,916 @@
+
+{% highlight xaml %}
+<ContentPage.Resources>
+    <ResourceDictionary>
+        <local:DisplayBindingConverter x:Key="displayBindingConverter" />
+    </ResourceDictionary>
+</ContentPage.Resources> 
+
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                       ItemsSource="{Binding OrdersInfo}">
+
+    <syncfusion:SfDataGrid.Columns>
+        <syncfusion:DataGridTextColumn MappingName="CustomerID" 
+                                   DisplayBinding="{Binding CustomerID, 
+                                   Converter={StaticResource displayBindingConverter}}" />
+    </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid> 
+{% endhighlight %}
+
+{% highlight c# %}
+public class DisplayBindingConverter : IValueConverter
+{
+    public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        if (value != null)
+            return "Customer:" + value.ToString();
+        return null;
+    }
+
+    public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+    {
+        return value.ToString().Substring(9);
+    }
+}
+{% endhighlight %}
+
+### Header customizations
+
+#### HeaderText
+
+To customize the display content of the header cell, use the `DataGridColumn.HeaderText` property. It specifies the text displayed in the column header. If header text is not defined, then `DataGridColumn.MappingName` will be assigned to the header text and will be displayed as column header.
+
+#### HeaderTextAlignment
+
+To get or set the TextAlignment of the header cell, use the `DataGridColumn.HeaderTextAlignment` property. The default alignment is `Center`. It can be customized as `Start` or `End`.
+
+#### HeaderTemplate
+
+Based on the requirement, the header cell can be customized using the `DataGridColumn.HeaderTemplate` property. To customize header cell by loading a template in the header cell, follow the code example:
+
+{% highlight xaml %}
+<syncfusion:SfDataGrid.Columns>
+    <syncfusion:DataGridTextColumn="OrderID">
+        <syncfusion:DataGridTextColumn.HeaderTemplate>
+            <DataTemplate>
+                <Label x:Name="OrderID" Text="OrderID" TextColor="Black" 
+                       BackgroundColor="Yellow"/>
+            </DataTemplate>
+        </syncfusion:DataGridTextColumn.HeaderTemplate>
+    </syncfusion:DataGridTextColumn>
+</syncfusion:SfDataGrid.Columns> 
+{% endhighlight %}
+
+### Setting manual column width
+
+SfDataGrid allows you to customize the width of each DataGridColumn in the `SfDataGrid.Columns` collection. To customize column width, use the `DataGridColumn.Width` property. By default, this property will not be assigned any value. The DataGridColumn renders in view based on the value of the `DefaultColumnWidth` property.
+
+N> Set the `Visible` property to `True` instead of setting column width as `0` to hide a column.
```

Comment:
N> Set the `Visible` property to `True` instead of setting the column width as `0` to hide a column.